contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -629,16 +629,14 @@ func validateLeadershipTransition(desired, current string) {
 }
 
 // hacked from https://github.com/GoogleCloudPlatform/kubernetes/blob/release-0.14/cmd/kube-apiserver/app/server.go
-func newEtcd(etcdConfigFile string, etcdServerList []string) (client *etcd.Client, err error) {
-	if etcdConfigFile != "" {
-		client, err = etcd.NewClientFromFile(etcdConfigFile)
-	} else {
-		client = etcd.NewClient(etcdServerList)
+func newEtcd(etcdServerList []string) (etcd.Client, error) {
```

Comment:
so what's the recommended way to connect securely to etcd right now? we should surface that somewhere if we haven't already, I see a good amount of confusion on slack.